### ISSUE TITLE
Question about mini-batch GCN training

### ISSUE THREAD

**Issue by IaroslavElistratov** [rusty1s/pytorch_geometric#2354]
## ❓ Questions & Help

Hi! When running [this script](https://github.com/snap-stanford/ogb/blob/master/examples/nodeproppred/products/cluster_gcn.py) `approximate train accuracy` seem to be about 50% lower than accuracy during inference -- I'd appreciate if you could give a pointer as to why this might happen. Thank you!

![Screen Shot 2021-04-06 at 10 39 10 PM](https://user-images.githubusercontent.com/57019863/113771116-aa363800-972b-11eb-8585-34ea93f3576a.png)

**Comment by rusty1s** [comment]
The `Approx Train Acc` computes training performance during sampling, i.e. it only operates on the clustered subgraphs instead of on the full graph. Not quite sure why the gap is so big, but it highlights the need for running Cluster-GCN on the full graph during inference.  Furthermore, since the model is in training mode, the usage of dropout may weaken the training performance.
